source/reference/minio-mc/mc-support-perf.rst

```diff
@@ -105,6 +109,16 @@ Run a network throughput test on a cluster with alias ``minio1``.
 
    mc support perf net minio1
 
+Test Site Replication Speed
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run a test on the speed of site replication operations across peers for the peer at ``minio1``.
```

Comment:
I'm having trouble following this.

Does this test *all* peers in the configuration, or just connections between `minio1` and other peers?

Reply:
Reworked to address.